translations/ja.json

```
@@ -160,48 +160,48 @@
 	"Docker Desktop only has {{.size}}MiB available, you may encounter application deployment failures.": "Docker Desktop では {{.size}}MiB しか利用できないため、アプリケーションのデプロイに失敗することがあります。",
 	"Docker container exited prematurely after it was created, consider investigating Docker's performance/health.": "Docker コンテナーは、作成後に途中で終了しました。Docker の動作や状態の調査を検討してください。",
 	"Docker has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "Docker は 2 つ未満の CPU で利用可能ですが、Kubernetes では少なくとも 2 つ必要です",
-	"Docker inside the VM is unavailable. Try running 'minikube delete' to reset the VM.": "",
-	"Docs have been saved at - {{.path}}": "ドキュメントは以下のパスに保存されました。{{.path}}",
+	"Docker inside the VM is unavailable. Try running 'minikube delete' to reset the VM.": "VM 内の Docker が利用できません。'minikube delete' を実行して、VM を初期化してみてください。",
+	"Docs have been saved at - {{.path}}": "ドキュメントは次のパスに保存されました - {{.path}}",
 	"Documentation: {{.url}}": "ドキュメント: {{.url}}",
-	"Done! kubectl is now configured to use \"{{.name}}\"": "完了しました！ kubectl が「\"{{.name}}\"」を使用するよう構成されました",
-	"Done! kubectl is now configured to use \"{{.name}}\" cluster and \"{{.ns}}\" namespace by default": "完了しました！ kubectl が「\"{{.name}}\"」クラスタと「\"{{.ns}}\"」ネームスペースを使用するよう構成されました",
-	"Done! minikube is ready without Kubernetes!": "",
-	"Download complete!": "ダウンロードが完了しました",
-	"Downloading Kubernetes {{.version}} preload ...": "Kubernetes {{.version}} のダウンロードの準備をしています",
+	"Done! kubectl is now configured to use \"{{.name}}\"": "終了しました! kubectl が「{{.name}}」を使用するよう設定されました",
+	"Done! kubectl is now configured to use \"{{.name}}\" cluster and \"{{.ns}}\" namespace by default": "終了しました!  kubectl がデフォルトで「{{.name}}」クラスターと「{{.ns}}」ネームスペースを使用するよう設定されました",
```

```suggestion
	"Done! kubectl is now configured to use \"{{.name}}\" cluster and \"{{.ns}}\" namespace by default": "終了しました! kubectl がデフォルトで「{{.name}}」クラスターと「{{.ns}}」ネームスペースを使用するよう設定されました",
```
There is an extra blank.

---

done.